src/runtime/mod.rs

```diff
@@ -18,6 +18,18 @@ use crate::{
     options::StreamAddress,
 };
 
+#[cfg(feature = "tokio-runtime")]
+lazy_static::lazy_static! {
+    static ref SYNC_RUNTIME: std::sync::Mutex<tokio::runtime::Runtime> = {
```

Comment:
Calling this `SYNC_RUNTIME` feels a little weird; I know that it's only used in the sync API, but it sounds like it's saying its a synchronous runtime, which it's not. Maybe we could call this `GLOBAL_RUNTIME`?

Reply:
Works for me!